examples/cactus-example-carbon-accounting-backend/package.json

```diff
@@ -14,6 +14,8 @@
   "bin": "./dist/lib/main/typescript/carbon-accounting-app-cli.js",
   "scripts": {
     "tsc": "tsc --project ./tsconfig.json",
+    "build:dev" : "npm run tsc || true && cp -r src/utility-emissions-channel dist/lib/",
```

Comment:
@Zzocker Is this meant to be executed inside the package directory or from the project root through lerna?

Reply:
it should be executed inside package directory.

Reply:
@Zzocker And this works fine? E.g. `cd ./examples/cactus-example-carbon-accounting-backend/ && npm run build:dev` doesn't crash? If yes, that's cool, I'm just wondering, because I would've expected this to not work due to how lerna symlinks the common dependencies.

Reply:
sorry , I forgot cactus uses lerna for building multiple packages present in cactus. 
For running the carbon accounting backend :
```
npm run build:dev:example-carbon-accounting-backend
npm run start:example-carbon-accounting
```
from project root.

Reply:
@Zzocker No worries, looks good now with the changes. 
FYI: We have a show-stopper bug on the main branch that is making all the tests fail for all the PRs so it will be a minute before this one is possible to be merged, hang in there and keep an eye out for this one https://github.com/hyperledger/cactus/pull/1073 being merged (after which you can rebase onto upstream/main and your tests should start passing for this PR at the same time)